index.html

```diff
@@ -0,0 +1,1323 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Fontawesome -->
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.8.1/css/all.css"
+      integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" type="text/css" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="shortcut icon" type="image/png" href="https://g.newsweek.com/themes/newsweek/favicons/favicon-96x96.png" />
+    <title>Newsweek Clone</title>
+  </head>
+  <body>
+    <div class="container">
+      <header class="contanier-fluid ">
+        <div class="row justify-content-center align-items-end logo-area ">
+          <div class="col d-none d-md-block mx-3 pb-4"><span class="current-date">Mon, November 04, 2019</span></div>
+          <div class="col-4 col-md-4 img-logo"><a href="#"><img src="images/newsweeklogo.png" class="img-fluid"
+                alt="newsweek logo"></a></div>
+          <div class="col btn-col">
+            <button type="button" class="btn trans-btn d-block d-xl-none float-right pb-3"><i class="fa fa-bars" aria-hidden="true"></i></button>
+            <button type="button" class="btn float-right subscribe-btn text-danger bg-white mx-3 mb-4 pb-1">SUBSCRIBE <i class="fas fa-chevron-right"></i></button>
+            <button type="button" class="btn float-right mr-2 signin-btn text-white pb-4">SIGN IN</button>
+      
+          </div>
+        </div>
```

Comment:
The width of the navbar on all screens is inaccurate. Please see the images below:

**Yours**
![navbar](https://user-images.githubusercontent.com/31304943/69074013-d5cd5b00-0a2e-11ea-8705-f13ddb3559d6.png)
![navbar2](https://user-images.githubusercontent.com/31304943/69074015-d5cd5b00-0a2e-11ea-87fa-0695ab5bd97b.png)
![navbar3](https://user-images.githubusercontent.com/31304943/69074016-d665f180-0a2e-11ea-9078-0838084b9568.png)

**Original**
![nav-def](https://user-images.githubusercontent.com/31304943/69074256-4aa09500-0a2f-11ea-86a0-b846c393591a.png)
![nav-def-2](https://user-images.githubusercontent.com/31304943/69074258-4b392b80-0a2f-11ea-9b6a-210ce3ada5e9.png)
![nav-def-3](https://user-images.githubusercontent.com/31304943/69074260-4b392b80-0a2f-11ea-8aef-535ccc25409a.png)

Reply:
The navigation links/buttons alignments are also inaccurate on a medium screen.